docs/structure.md

```diff
@@ -22,25 +22,28 @@ function.
 !!! example "Available Functionality"
     === "up42"
     
-        - `.initialize_project()`
-        - `.initalize_workflow()`
-        - `.initalize_job()`
-        - `.initalize_jobtask()`
-        - `.initialize_jobcollection()`
-        - `.initalize_catalog()`
-        - `.initalize_storage()`
-        - `.initalize_order()`
-        - `.initalize_asset()`
+        up42 is the base object imported to Python. It provides the elementary functionality 
```

Comment:
up42 is a module I think, not an object. I've also noticed that we used the term object often, which technically is incorrect. Project is a class, when you instantiate a Project you get an object. In any case everything is an object in python so I guess it's fine.

Reply:
Yeah, describing it as objects was a conscious descision because I wanted to avoid that people think they need to deal with Python classes, object is neutral. Will adjust to class, I don't see it so critical anymore.